modules/tensor_mechanics/doc/content/source/materials/ComputePlaneSmallStrain.md

```diff
@@ -117,11 +125,27 @@ behind [eqn:vlc_strain] see the
 [Volumetric Locking Correction](/tensor_mechanics/VolumetricLocking.md)
 documentation.
 
-## Example Input File
+## Example Input Files
+
+### Plane Stress
+
+The tensor mechanics [MasterAction](/Modules/TensorMechanics/Master/index.md)
```

Comment:
```suggestion
The tensor mechanics [Master action](/Modules/TensorMechanics/Master/index.md)
```

`MasterAction` is neither a thing in the Syntax, nor in the Source.

Reply:
Fixed